cnf-certification-test/webserver/index.html

```diff
@@ -270,43 +275,70 @@ <h1>CNF Certification Test</h1>
         
         </fieldset>
       </div>
-
         <fieldset>
           <legend>Select a Test</legend>
-          <label>
+
+          <legend id="selectComboBox"> Selcet a Test Tag <select name="class" id="selectScenarioComboBox" onchange="selectScenario()">
+          
+            <option value="all"> All</option>
+            <option value="faredge"> Far-Edge</option>
+            <option value="telco"> Telco</option>
+            <option value="nontelco"> Non-Telco</option>
+            <option value="extended"> Extended</option>
+
+          </select>
+          <select name="class" id="selectOpt" onchange="selectScenario()" hidden>
+            <option value="Mandatory"> Mandatory</option>
+            <option value="Optional"> Optional</option>
+         
+          </select>
+
+        </legend>
+          <label id="selectedsuites" hidden>
```

Comment:
The list of suites could be obtained from the classification.json after adding a suite name to it. Then we could just loop throught the suite names to generate the tabs/checkboxes, etc...